### PR TITLE
Update references to DiaSymreader in project.json and proj files

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@agocke @tannergooding

File: build/ToolsetPackages/project.lock.json [hunk 1]
```diff
@@ -29,12 +29,12 @@
           "lib/net45/Microsoft.CodeAnalysis.Test.Resources.Proprietary.dll": {}
         }
       },
-      "Microsoft.DiaSymReader/1.0.5": {
+      "Microsoft.DiaSymReader/1.0.5.1": {
         "compile": {
-          "lib/net45/Microsoft.DiaSymReader.dll": {}
+          "lib/net20/Microsoft.DiaSymReader.dll": {}
```
Comment (agocke) [inline build/ToolsetPackages/project.lock.json]: Why are we going backwards here? Why net20?

Comment (tmat) [inline build/ToolsetPackages/project.lock.json]: Hmm, i don't know.

Comment (tmat) [inline build/ToolsetPackages/project.lock.json]: The package has two targets:
net20
portable-net45+win8

I'm not sure why net20 was picked.

**Pull request closed without merging**